Makefile

```diff
@@ -2,18 +2,21 @@ usage:         ## Show this help
 	@fgrep -h "##" $(MAKEFILE_LIST) | fgrep -v fgrep | sed -e 's/\\$$//' | sed -e 's/##//'
 
 install:       ## Install dependencies for all projects
-	CMD='make install' make for-each-dir
+	MAKE_TARGET='install' make for-each-dir
 
 lint:          ## Run code linter for all projects
 	CMD='make lint' make for-each-dir
 
 start:         ## Start LocalStack infrastructure
 	nohup localstack start &
```

Comment:
not by me, I simply didn't remove them because this PR already touched too many things. Just in case some sample that doesn't have a (proper) Makefile yet can still get LS started somehow. Scrap?

Reply:
Adjusted as suggested